lib/serializers/serializerUtilities.js

```diff
@@ -0,0 +1,22 @@
+const getBibCallNum = (object, builder, fieldMapper, bib = true) => {
```

Comment:
I'd suggest documenting this method because the contract is confusing, but I also think you should change the contract. It mutates one of the arguments, but only some of the time.. It appears to work like this:
- Given a bib ("object"), identifies the call number in one of two places.
- If a call number is found it:
  - If `bib` is truthy, it adds the statement via the passed `builder` and returns the callnumber value
  - Otherwise, it just returns the callnumber value.

In general I'd avoid mutating args, and definately avoid doing so only some of the time.

Can this `getBibCallNum` just take the `bib` and return the call number (and the path it found it in?) I'd suggest a contract that was simpler: Just have it return an object like `{ value, path }`. That would require changing the bib serializer call to:
```
  const callnum = getBibCallNum(object)
  if (callnum) {
    builder.add(fieldMapper.predicateFor('Call number'), { literal: callnum.value }, 0, { path: callnum.path })
  }
```

And the item serializer call would be simplified to:
```
  const bibCallNum = getBibCallNum(bib)
```

Reply:
Refactored to match the suggestions above